src/main/webapp/app/shared/user-settings/user-settings-container/user-settings-container.component.ts

```diff
@@ -33,7 +37,16 @@ export class UserSettingsContainerComponent implements OnInit {
 
         this.accountService
             .getAuthenticationState()
-            .pipe(tap((user: User) => (this.currentUser = user)))
+            .pipe(
+                tap((user: User) => {
+                    this.currentUser = user;
+                    this.authorizeTutor();
```

Comment:
```suggestion
                    this.isAtLeastTutor = this.accountService.isAtLeastTutor();
```
see other comment below